test_conformance/subgroups/main.cpp

```diff
@@ -66,6 +68,22 @@ static test_status InitCL(cl_device_id device)
             ret = TEST_SKIP;
         }
     }
+    // Determine the rounding mode to be used in float to half conversions in
+    // init and reference code
+    const cl_device_fp_config fpConfig = get_default_rounding_mode(device);
+
+    if (fpConfig == CL_FP_ROUND_TO_NEAREST)
+    {
+        g_rounding_mode = CL_HALF_RTE;
+    }
+    else if (fpConfig == CL_FP_ROUND_TO_ZERO && gIsEmbedded)
```

Comment:
Sorry, missed the discussion previously on this point.

I don't think that `gIsEmbedded` is required, but works for us so wont let it block this PR.